mappings/net/minecraft/client/option/CyclingOption.mapping

```diff
@@ -22,25 +29,34 @@ CLASS net/minecraft/class_4064 net/minecraft/client/option/CyclingOption
 		ARG 2 setter
 	METHOD method_32523 create (Ljava/lang/String;Ljava/util/function/Supplier;Ljava/util/function/Function;Ljava/util/function/Function;Lnet/minecraft/class_4064$class_5675;)Lnet/minecraft/class_4064;
 		ARG 0 key
+		ARG 1 valuesSupplier
+		ARG 2 valueToText
 		ARG 3 getter
 		ARG 4 setter
 	METHOD method_32524 create (Ljava/lang/String;Lnet/minecraft/class_2561;Ljava/util/function/Function;Lnet/minecraft/class_4064$class_5675;)Lnet/minecraft/class_4064;
 		ARG 0 key
+		ARG 1 tooltip
 		ARG 2 getter
 		ARG 3 setter
 	METHOD method_32525 create (Ljava/lang/String;Lnet/minecraft/class_2561;Lnet/minecraft/class_2561;Ljava/util/function/Function;Lnet/minecraft/class_4064$class_5675;)Lnet/minecraft/class_4064;
 		ARG 0 key
+		ARG 1 on
+		ARG 2 off
 		ARG 3 getter
 		ARG 4 setter
 	METHOD method_32526 create (Ljava/lang/String;[Ljava/lang/Object;Ljava/util/function/Function;Ljava/util/function/Function;Lnet/minecraft/class_4064$class_5675;)Lnet/minecraft/class_4064;
 		ARG 0 key
+		ARG 1 values
+		ARG 2 valueToText
 		ARG 3 getter
 		ARG 4 setter
 	METHOD method_32527 (Ljava/util/List;Ljava/lang/Boolean;)Ljava/util/List;
 		ARG 1 value
+	METHOD method_32528 tooltip (Ljava/util/function/Function;)Lnet/minecraft/class_4064;
+		ARG 1 tooltips
 	METHOD method_32532 (Lnet/minecraft/class_2561;Lnet/minecraft/class_310;)Lnet/minecraft/class_5676$class_5679;
 		ARG 1 client
-	CLASS class_5675
+	CLASS class_5675 Consumer
```

Comment:
```suggestion
	CLASS class_5675 Setter
```
This interface is used exclusively for setters.

Reply:
actually you are right, this name is more concise as well!